generators/base/logic/connect.mjs

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I don't like the folder called `logic` I prefer `support`.
Please open a issue to discuss.

Reply:
support is less clear to me, still, the issue is here: https://github.com/jhipster/generator-jhipster/issues/20295